Source/Diagnostics/ReducedDiags/FieldProbe.cpp

```diff
@@ -164,6 +164,7 @@ FieldProbe::FieldProbe (std::string rd_name)
 
             // write header row
             int c = 0;
+            ofs << "#";
```

Comment:
```suggestion
            // No "#" symbol in FieldProbe header, because the recommended way to read FieldProbe
            // output is to use pandas, which requires reading the header rather than treating it as a
            // commented line.
            // ofs << "#";
```